docs/running_psalm/issues/PropertyNotSetInConstructor.md

```diff
@@ -1,6 +1,16 @@
 # PropertyNotSetInConstructor
 
-Emitted when a non-null property without a default value is declared but not set in the class’s constructor
+Unitialized properties are statically hard to analyze. To prevent mistakes, Psalm will enforce that all properties should be initialized.
```

Comment:
```suggestion
Unitialized properties are hard to statically analyze. To prevent mistakes, Psalm will enforce that all properties should be initialized.
```